pkg/ddevapp/templates.go

```diff
@@ -98,12 +98,17 @@ const ConfigInstructions = `
 # database container will be unusable. In the global configuration it is also
 # possible to omit ddev-router, but not here.
 
-# nfs_mount_enabled: false
-# Great performance improvement but requires host configuration first.
+# performance: ""
```

Comment:
Let's not use empty.

Reply:
Empty will omit the value from the config which is what I wanted to achieve here. Alternatively we can use `default`` but I really prefer to omit the value here.